nameable.rb

```diff
@@ -0,0 +1,33 @@
+# Base class for decorator pattern
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you add the following commands in your Getting Started section to make your readme more descriptive and professional 👍

* bundle install
* ruby main.rb
* rspec

Reply:
Thank you for your time and attention. I will take care of it.